src/components/DialogBox/styles.scss

```diff
@@ -118,7 +118,6 @@
     border-bottom: 1px solid $tw-clr-brdr-tertiary;
     padding: $tw-spc-base-medium;
     text-align: center;
-    overflow: scroll;
```

Comment:
Oops, missed this earlier change.. thanks for catching. Scrolled=false allows us to show dialog boxes with dropdowns that don't extend into the dialog body, instead extend outside of the dialog.